client/src/models/AddReview.ts

```diff
@@ -10,6 +11,9 @@ const AddReview = z.object({
     idBook: z.custom<BookViewModelType>()
         .refine((book) => book != null, "Pole wymagane")
         .transform(book => book.id),
+    idUser: z.custom<UserDetailViewModelType>()
+    .refine((user) => user != null, "Pole wymagane")
+    .transform(user => user.id),
```

Comment:
dalej chyba nie masz tego prettiera albo nie masz go włączonego

Reply:
eh mam ale sprawdze to